backend/molgenis-emx2/src/main/java/org/molgenis/emx2/utils/JavaScriptUtils.java

```diff
@@ -7,22 +7,22 @@
 import org.graalvm.polyglot.Value;
 import org.molgenis.emx2.MolgenisException;
 import org.molgenis.emx2.utils.generator.IdGenerator;
+import org.molgenis.emx2.utils.generator.IdGeneratorImpl;
 
 public class JavaScriptUtils {
+  public static IdGenerator idGenerator = new IdGeneratorImpl();
```

Comment:
this way there is no way of knowing what the impl is for the id genarator when it is called via JavaScriptUtils nice for testing, but hard to reason and maintain. please make it private again ( and inject the impl via the constructor)